app/src/features/surveys/observations/observations-table/configure-columns/components/environment/search/EnvironmentsSearchAutocomplete.tsx

```diff
@@ -69,6 +70,20 @@ export const EnvironmentsSearchAutocomplete = (props: IEnvironmentsSearchAutocom
     [getOptions]
   );
 
+  const loadAllOptions = async () => {
+    const response = await getOptions('');
+    const allOptions = [...response.qualitative_environments, ...response.quantitative_environments];
+
+    const sortedOptions = allOptions.sort((a, b) => a.name.localeCompare(b.name));
+    setOptions(sortedOptions);
+  };
+
+  // This function is a repeat of a function in another file for pluralising. It should be edited to an imported function. GridColumnDefinitionsUtils.tsx when PR goes through
+  const getFormattedUnit = (unit: string) => {
+    const unitSuffix = unit.endsWith('er') ? 's' : '';
```

Comment:
I think this is the second time we've seen this logic. I would suggest making it a standalone function.

I would add a new function next to the `pluralize` function in `.../utils/Utils.tsx` since it tackles a similar kind of issue.